### PR TITLE
 [Google Drive] Add an action to toggle right-side details view 

### PR DESCRIPTION
**Pull request by vishaltelangre**
## Screencast

image

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/google-drive/src/index.tsx [hunk 1]
```diff
@@ -262,36 +258,38 @@ const ListItem = ({ file, idPrefix }: ListItemProps) => {
...
-      icon={Icon.Hammer}
-      onAction={reindexFiles}
-      shortcut={{ modifiers: ["cmd", "shift"], key: "r" }}
-    />
+    <ActionPanel.Section title="General Actions">
+      <Action
+        title="Toggle Details"
+        icon={Icon.Sidebar}
+        onAction={toggleDetails}
+        shortcut={{ modifiers: ["cmd"], key: "b" }}
```
Comment (vishaltelangre) [inline extensions/google-drive/src/index.tsx:277]: This keybinding is inspired by VSCode to toggle show/hide the sidebar.

File: extensions/google-drive/src/index.tsx [hunk 2]
```diff
@@ -237,7 +232,8 @@ const ListItem = ({ file, idPrefix }: ListItemProps) => {
       id={`${idPrefix}${file.displayPath}`}
       key={file.displayPath}
       icon={{ fileIcon: file.path }}
-      title={`${file.favorite ? "⭐ " : ""}${file.name}`}
+      title={file.name}
+      accessoryIcon={file.favorite ? "⭐" : undefined}
```
Comment (vishaltelangre) [inline extensions/google-drive/src/index.tsx:236]: This looks better! See the screenshot in the PR's description.

File: extensions/google-drive/src/index.tsx [hunk 3]
```diff
@@ -262,36 +258,40 @@ const ListItem = ({ file, idPrefix }: ListItemProps) => {
...
-      onAction={reindexFiles}
-      shortcut={{ modifiers: ["cmd", "shift"], key: "r" }}
-    />
+    <ActionPanel.Section title="General Actions">
+      {showToggleDetailsAction ? (
+        <Action
+          title="Toggle Details"
+          icon={Icon.Sidebar}
+          onAction={toggleDetails}
+          shortcut={{ modifiers: ["cmd"], key: "b" }}
```
Comment (vishaltelangre) [inline extensions/google-drive/src/index.tsx:278]: This keybinding is inspired by VSCode to toggle show/hide the sidebar.

**Comment by vishaltelangre** [comment]
@pernielsentikaer Please review.

**Review by pernielsentikaer — APPROVED** [review]
Awesome @vishaltelangre 🔥 Another great update, thanks 💪

**Pull request merged**